metrics/web/test/dashboard/presentation/state/project_metrics_notifier_test.dart

```diff
@@ -158,13 +183,17 @@ void main() {
     );
 
     test(
-      "creates ProjectMetricsData with null metrics if the DashboardProjectMetrics is null",
+      "creates project metrics tile view models with null metrics if the emitted DashboardProjectMetrics and BuildDayProjectMetrics are null",
```

Comment:
I think it would be great to divide this test case into 2 test cases - the case when the emitted `DashboardProjectMetrics` is null and when the emitted `BuildDayProjectMetrics` is null.